pkg/validator/env.go

```diff
@@ -0,0 +1,22 @@
+package validator
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/hashicorp/go-envparse"
+)
+
+type EnvValidator struct{}
+
+// Validate implements the Validator interface by attempting to
+// parse a byte array of a env file using envparse package
+func (envv EnvValidator) Validate(b []byte) (bool, error) {
+	r := bytes.NewReader(b)
+	_, err := envparse.Parse(r)
+	if err != nil {
+		customError := err.(*envparse.ParseError)
+		return false, fmt.Errorf("Error at line %v: %v", customError.Line, customError.Err)
```

Comment:
I think evsparse package is not suporting it. If ir is necessary we need to write our own. I thought it would be okay because Env is simpler type than the others.

Reply:
Yeah it's not a problem if it doesn't support it. File syntax is simple
Enough just the line should be good enough. Will mark as resolved

Reply:
Ok, thanks